openapi/listings/listings.yaml

```diff
@@ -401,6 +401,9 @@ components:
           description: Vendasta's unique ID for the business
         type:
           type: string
+          default: listingScore
```

Comment:
Again this should be plural.

Reply:
It's a get endpoint tho, and there is only 1 score per business

Reply:
On the other hand we do offer a breakdown of the score.